pkg/rtc/transport/webrtctransport.go

```diff
@@ -236,9 +236,10 @@ func NewWebRTCTransport(id string, options RTCOptions) *WebRTCTransport {
 	w.pc.OnICEConnectionStateChange(func(connectionState webrtc.ICEConnectionState) {
 		switch connectionState {
 		case webrtc.ICEConnectionStateDisconnected:
-			log.Errorf("webrtc ice disconnected")
+			log.Infof("webrtc ice disconnected for mid: %s", id)
 		case webrtc.ICEConnectionStateFailed:
-			log.Errorf("webrtc ice failed")
+		case webrtc.ICEConnectionStateClosed:
```

Comment:
Should we handle `Closed` the same way as `Failed`? I actually haven't seen `Closed` triggered when doing `pc.Close()` with a pion client. Not sure if i am misunderstanding.